packages/node/src/indexer/dictionary.service.ts

```diff
@@ -16,4 +22,21 @@ export class DictionaryService
   constructor(protected project: SubqueryProject, nodeConfig: NodeConfig) {
     super(project.network.dictionary, nodeConfig);
   }
+
+  async getEvmChainId(): Promise<string> {
+    const query = `query{evmChainAlias(id: "chainId"){value}}`;
```

Comment:
I think this should be
```suggestion
    const query = `query{chainAlias(id: "evmChainId"){value}}`;
```